package.json

```diff
@@ -38,7 +38,7 @@
     "coverage": "jest --coverage",
     "ci:publish": "yarn build && yarn semantic-release && yarn clean",
     "semantic-release": "semantic-release",
-    "clean": "rm -rf ./build && rm ./boilerplate/.gitignore.template",
+    "clean": "rm -rf ./build ./boilerplate/.gitignore.template ./boilerplate/node_modules ./boilerplate/ios/build/* ./boilerplate/ios/Pods/* ./boilerplate/android/app/build ./boilerplate/ios/.xcode.env.local",
```

Comment:
We will want to update this script to not throw an error if these files 
```suggestion
    "clean": "rm -drf ./build ./boilerplate/.gitignore.template ./boilerplate/node_modules ./boilerplate/ios/build/ ./boilerplate/ios/Pods/ ./boilerplate/android/app/build ./boilerplate/ios/.xcode.env.local",
```

<img width="556" alt="Screen Shot 2022-09-20 at 10 19 33 AM" src="https://user-images.githubusercontent.com/37849890/191323251-1b71027d-a1ff-42f0-bfd7-d5d4b2ee93a7.png">